tests/installation/installation_settings/validate_default_target.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The test module goes to install-shell to validate if the actual
+# systemd target corresponds to the expected one.
+# Then returns back to the installation console.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use Test::Assert 'assert_equals';
+
+sub run {
+    my ($self) = @_;
+    my $expected_target = check_var('DESKTOP', 'textmode') ? "multi-user" : "graphical";
```

Comment:
As I stated above, I assume having this as a function in `y2_installbase` as a temporary solution. When we move all the test to libyuiclient, we'll move the contents of the function to `validate_default_target.pm` and it will be only a single place then.

And then `$expected_target` will be just a pure test data, so it should be specified on Test Module layer. 

Is it ok, WDYT?